docs/api.md

```diff
@@ -0,0 +1,103 @@
+---
+sidebar_position: 6
+---
+# API Usage
+
+## Postman
+
+A collection of OpenCost Postman queries: [opencost.postman_collection.json](https://raw.githubusercontent.com/kubecost/opencost/develop/docs/opencost.postman_collection.json)
+
+> Note: Change the hostname in the Collection>Edit>Variables
+
+## Usage
+
+```
+GET /allocation/compute
```

Comment:
should we document other APIs that are exposed?

Reply:
I'll lean on @michaelmdresser for this one. I heard costDataModel is deprecated?

Reply:
`costDataModel` is definitely deprecated and I think we should actively discourage its use. I'm not sure what other APIs we expose, but I suspect that (at the moment) `/allocation/compute` is the only one that _must_ be documented.